2025-09-07 17:58:57,584 : INFO : root : Loading config.
2025-09-07 17:58:57,589 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2025-09-07 17:58:57,589 : INFO : ConfigProcessor : Loading config from cache: C:\Users\White\AppData\Local\Temp\cfa6ecba65704a931c9dd214c44986c8.mpf_cache
2025-09-07 17:58:57,702 : INFO : ConfigProcessor : Loading config from cache: C:\Users\White\AppData\Local\Temp\f98ffc0ff5962effface7d86bc406d5c.mpf_cache
2025-09-07 17:58:57,703 : INFO : ConfigProcessor : Loading config from cache: C:\Users\White\AppData\Local\Temp\91cc4d1c993c58dc354e34974b9cfeac.mpf_cache
2025-09-07 17:58:57,703 : INFO : ConfigProcessor : Loading config from cache: C:\Users\White\AppData\Local\Temp\f98ffc0ff5962effface7d86bc406d5c.mpf_cache
2025-09-07 17:58:57,704 : INFO : ConfigProcessor : Loading config from cache: C:\Users\White\AppData\Local\Temp\76792f1d686f89709887dba7bd69667a.mpf_cache
2025-09-07 17:58:57,705 : INFO : ConfigProcessor : Loading config from cache: C:\Users\White\AppData\Local\Temp\142bbae4707335cc8e3efaa4364c4893.mpf_cache
2025-09-07 17:58:57,705 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev9
2025-09-07 17:58:57,706 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': True, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs\\2025-09-07-17-58-57-mpf-TopGunPinball.log', 'pause': False, 'production': False, 'text_ui': False, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': 'virtual', 'syslog_address': None, 'full_logfile_path': 'C:\\Users\\White\\Documents\\Top_Gun_Pinball\\logs\\2025-09-07-17-58-57-mpf-TopGunPinball.log'}
2025-09-07 17:58:57,706 : INFO : Machine : MPF path: C:\Users\White\mpf\Lib\site-packages\mpf
2025-09-07 17:58:57,706 : INFO : Machine : Machine path: C:\Users\White\Documents\Top_Gun_Pinball
2025-09-07 17:58:57,706 : INFO : Machine : Platform: win32
2025-09-07 17:58:57,706 : INFO : Machine : Python executable location: C:\Users\White\mpf\Scripts\python.exe
2025-09-07 17:58:57,706 : INFO : Machine : Python version: 3.12.7 (64-bit)
2025-09-07 17:58:57,707 : INFO : Machine : Initializing MPF...
2025-09-07 17:58:57,796 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,823 : INFO : EventManager : Event: ======'machine_var_master_volume'====== Args={'value': 0.5, 'prev_value': None, 'change': True}
2025-09-07 17:58:57,823 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 61000, 'prev_value': None, 'change': True}
2025-09-07 17:58:57,823 : INFO : EventManager : Event: ======'machine_var_player2_score'====== Args={'value': 73000, 'prev_value': None, 'change': True}
2025-09-07 17:58:57,823 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev9', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,823 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev9, Config version:6, Show version: 6, BCP version:1.1', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,824 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.12.7', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,904 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-11-10.0.26100-SP0', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,905 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,905 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '11', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,905 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '10.0.26100', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,905 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,905 : INFO : EventManager : Event: ======'machine_var_log_file_path'====== Args={'value': 'C:\\Users\\White\\Documents\\Top_Gun_Pinball\\logs\\2025-09-07-17-58-57-mpf-TopGunPinball.log', 'prev_value': None, 'change': True}
2025-09-07 17:58:57,905 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2025-09-07 17:58:57,929 : ERROR : ConfigValidator : Your config contains a value for the setting "playfields:playfield:enable_switches", but this is not a valid setting name.
2025-09-07 17:58:57,929 : ERROR : asyncio : Exception in callback EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...tting name.')>) at C:\Users\White\mpf\Lib\site-packages\mpf\core\events.py:124
handle: <Handle EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...tting name.')>) at C:\Users\White\mpf\Lib\site-packages\mpf\core\events.py:124>
Traceback (most recent call last):
  File "C:\Users\White\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\events.py", line 127, in _async_handler_done
    future.result()
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\device_manager.py", line 88, in _load_device_modules
    self.load_devices_config(validate=True)
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\device_manager.py", line 146, in load_devices_config
    config[device_name] = collection[device_name].validate_and_parse_config(config[device_name], False)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\device.py", line 108, in validate_and_parse_config
    self.machine.config_validator.validate_config(
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\config_validator.py", line 164, in validate_config
    return self._validate_config(config_spec, source, base_spec=base_spec, add_missing_keys=add_missing_keys,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\config_validator.py", line 177, in _validate_config
    self.check_for_invalid_sections(this_spec, source, validation_failure_info)
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\config_validator.py", line 345, in check_for_invalid_sections
    raise ConfigFileError('Your config contains a value for the '
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigValidator: Your config contains a value for the setting "playfields:playfield:enable_switches", but this is not a valid setting name. Error Code: CFE-ConfigValidator-2 (https://missionpinball.org/logs)
2025-09-07 17:58:57,933 : INFO : Machine : Starting the main run loop with active modes: []
2025-09-07 17:58:57,933 : WARNING : Machine : Attract mode is not active, game will not be playable. Please check your attract mode configuration.
2025-09-07 17:58:57,934 : INFO : Machine : Shutting down...
2025-09-07 17:58:57,934 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2025-09-07 17:58:58,943 : ERROR : Machine : Runtime Exception
Traceback (most recent call last):
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\machine.py", line 808, in _run_loop
    raise self._exception['exception']
  File "C:\Users\White\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\events.py", line 127, in _async_handler_done
    future.result()
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\device_manager.py", line 88, in _load_device_modules
    self.load_devices_config(validate=True)
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\device_manager.py", line 146, in load_devices_config
    config[device_name] = collection[device_name].validate_and_parse_config(config[device_name], False)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\device.py", line 108, in validate_and_parse_config
    self.machine.config_validator.validate_config(
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\config_validator.py", line 164, in validate_config
    return self._validate_config(config_spec, source, base_spec=base_spec, add_missing_keys=add_missing_keys,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\config_validator.py", line 177, in _validate_config
    self.check_for_invalid_sections(this_spec, source, validation_failure_info)
  File "C:\Users\White\mpf\Lib\site-packages\mpf\core\config_validator.py", line 345, in check_for_invalid_sections
    raise ConfigFileError('Your config contains a value for the '
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigValidator: Your config contains a value for the setting "playfields:playfield:enable_switches", but this is not a valid setting name. Error Code: CFE-ConfigValidator-2 (https://missionpinball.org/logs)
2025-09-07 17:58:58,945 : INFO : root : MPF run loop ended.
